common/Projectile.js

```diff
@@ -29,7 +30,8 @@ Projectile.protocol = {
     speed: {type: nengi.Number, interp: false },
     width: { type: nengi.Number, interp: false },
     height: { type: nengi.Number, interp: false },
-    lifetime: {type: nengi.Number, interp: false}
+    lifetime: {type: nengi.Number, interp: false},
+    owner: {type: nengi.Number, interp: false }
```

Comment:
similar to the last PR, do you think the `owner` property is necessary to communicate to all observers of this entity on their clients?

i can think of one use case for sure - can you?